tests/cloudpickle_test.py

```diff
@@ -1119,6 +1119,20 @@ def func(x):
         cloned = pickle_depickle(func, protocol=self.protocol)
         self.assertEqual(cloned.__qualname__, func.__qualname__)
 
+    def test_property(self):
+        class MyObject:
+            _value = 1
+
+            @property
+            def value(self):
+                return self._value
+
```

Comment:
In addition to the read-only property, this class should have a read-write property and one should check that the reconstructed instance allows to set the value of the read-write property while it should not be possible for the read-only property of the reconstructed instance.

Reply:
Also the pickling of the `__doc__` info is not tested either.